### PR TITLE
Feature/public key hash

### PR DESCRIPTION
**Pull request by rober12**
- Updated the examples of the identity creation functions to use the new ones that works with the hash of the public key.
- Updated the examples of the public Key functions to use the new ones that works with the hash of the public key.
- Updated ci.yml to remove the step of executing the examples, since there is now no test blockchain node available and update the node version.

### CODE REVIEW

File: exampleFirstEntity/1.createEntityAlastriaID.js [hunk 1]
```diff
@@ -77,9 +77,11 @@ function preparedAlastriaId() {
 }
 
 function createAlastriaId() {
-  const txCreateAlastriaID = transactionFactory.identityManager.createAlastriaIdentity(
+  const entity1PubKeyHash = `${web3.utils.sha3(configData.entity1Pubk.substr(2))}`
+  console.log("PUBLIC KEY HASH", entity1PubKeyHash);
```
Comment (DaniRumpi) [inline exampleFirstEntity/1.createEntityAlastriaID.js]: quitaría este comentario, en los demás ejemplo no está

File: examplePublicKey/1.getSubject1CurrentPublicKey.js [hunk 2]
```diff
@@ -12,17 +12,25 @@ if (configData.didSubject1 === undefined) {
...
-
-const currentPubKey = transactionFactory.publicKeyRegistry.getCurrentPublicKey(
+const subject1PubKeyHash = `${web3.utils.sha3(configData.subject1Pubk.substr(2))}`
+const currentPubKey = transactionFactory.publicKeyRegistry.getPublicKeyStatusHash(
   web3,
-  configData.didSubject1
+  configData.didSubject1,
+  subject1PubKeyHash
 )
+console.log("TRANSACCION", currentPubKey);
```
Comment (DaniRumpi) [inline examplePublicKey/1.getSubject1CurrentPublicKey.js]: No se si este es necesario.

**Review by DaniRumpi — CHANGES_REQUESTED** [review]
Hay dos console.log que creo que sobran, míralos a ver que te parece.

File: exampleFirstEntity/1.createEntityAlastriaID.js (hunk 1 above)
Comment (rober12) [reply exampleFirstEntity/1.createEntityAlastriaID.js]: Done!

File: examplePublicKey/1.getSubject1CurrentPublicKey.js (hunk 2 above)
Comment (rober12) [reply examplePublicKey/1.getSubject1CurrentPublicKey.js]: Done!

**Review by DaniRumpi — APPROVED** [review]

**Pull request merged**